pgjdbc/src/test/java/org/postgresql/test/jdbc2/DriverTest.java

```diff
@@ -35,14 +35,27 @@
 import java.util.logging.Logger;
 
 /*
- * Tests the dynamically created class software.aws.rds.jdbc.postgresql.Driver
+ * Tests the dynamically created class org.postgresql.Driver
  *
  */
 public class DriverTest {
-  @BeforeEach
-  public void disableAcceptAwsProtocolOnly() {
-    TestUtil.initDriver(); // Set up log levels, etc.
-    Driver.setAcceptAwsProtocolOnly(false);
+
+  @Before
+  public void setUp() throws SQLException {
+
+    if (software.aws.rds.jdbc.postgresql.Driver.isRegistered()) {
```

Comment:
It shouldn't be references to AWS driver from this class.

Reply:
Sounds good. Removing this resulted in having to do this: [this](https://github.com/awslabs/aws-postgresql-jdbc/pull/39#discussion_r652122807)